.coveragerc

```diff
@@ -93,9 +93,7 @@ omit =
     homeassistant/components/*/pilight.py
 
     homeassistant/components/knx.py
-    homeassistant/components/switch/knx.py
-    homeassistant/components/binary_sensor/knx.py
-    homeassistant/components/thermostat/knx.py
+    homeassistant/components/*/knx.py
```

Comment:
That's fine - just always worry when I see this stuff. Like I said - not needed, just always a worry that something is broken.